pkg/telemetry/prometheus/rooms.go

```diff
@@ -10,22 +10,25 @@ import (
 )
 
 var (
-	roomTotal              atomic.Int32
-	participantTotal       atomic.Int32
-	trackPublishedTotal    atomic.Int32
-	trackSubscribedTotal   atomic.Int32
+	roomCurrent            atomic.Int32
```

Comment:
why not change `promRoomTotal` -> `proomRoomCurrent` as that is also decrementing on close?